console/src/components/Services/RemoteSchema/Edit/View.js

```diff
@@ -17,9 +17,9 @@ import ReloadRemoteSchema from '../../Settings/MetadataOptions/ReloadRemoteSchem
 
```

Comment:
I noticed that a part of this file uses `UNSAFE_componentWillReceiveProps` shall I move that code to `getDerivedStateFromProps` or any other permissible lifecycle method?////

Reply:
I have changed them in a couple of places relevant to this PR. There are 4 other such instances in the codebase, I can change those too and update them in the same PR or I could make another PR for those too